addons/web/static/src/views/record.js

```diff
@@ -44,6 +44,7 @@ class _Record extends Component {
         };
         onWillStart(() => load(this.props));
         onWillUpdateProps((nextProps) => {
+            this.model.rootParams.activeFields = this.getActiveFields(nextProps);
```

Comment:
can't you use a t-key in your templates instead of doing this. activeFields were never assumed to be variable

Reply:
I don't think the `t-key` will solve the issue. 
In fact, the record component is used to correctly display the row (and section) titles and those rows changed according to the groupby selected by the user with the search view and so the activeFields changed as well.

If you prefer, I could extend Record component to manage that case in the grid only. 🙂

Reply:
if you  use t-key correctly, a new component will be generated so that you won't have to change willUpdateProps.

Reply:
Oh cool, I didn't know that, I will try, thanks for the information. 🙂